### PR TITLE
props

### LINKED ISSUE #134: each kafka setting/property = object

### PR DESCRIPTION
**Pull request by l3r8yJ**
i think it's can close #134 

@h1alexbel take a look, please

### CODE REVIEW

**Comment by l3r8yJ** [comment]
@h1alexbel if it's okay, don't merge i'll add tests

File: src/main/java/io/github/eocqrs/kafka/settings/AttrEnvelope.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  public final String name() {
+    return this.name;
+  }
+
+  /**
+   * Name to camelcase.
+   *
+   * @return The name of the attribute as a tag.
+   */
+  private String nameInCamelCase() {
```
Comment (h1alexbel) [inline src/main/java/io/github/eocqrs/kafka/settings/AttrEnvelope.java:65]: Private method is untestable solution, we need introduce a new object called `CamelCasedText`.
Maybe we need even make a pr into cactoos library.
But for now, can you just add the puzzle about this?

**Review by h1alexbel — COMMENTED** [review]
@l3r8yJ check comments please

File: src/main/java/io/github/eocqrs/kafka/settings/AttrEnvelope.java (hunk 1 above)
Comment (l3r8yJ) [reply src/main/java/io/github/eocqrs/kafka/settings/AttrEnvelope.java:65]: @h1alexbel yes, sure i'll create a puzzle

File: src/main/java/io/github/eocqrs/kafka/settings/BootstrapServers.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+package io.github.eocqrs.kafka.settings;
+
+/**
+ * It's a wrapper for the `bootstrap.servers` kafka attribute.
+ *
+ * @author Ivan Ivanchuk (l3r8y@duck.com)
+ * @since 0.0.2
+ */
+public class BootstrapServers extends AttrEnvelope {
```
Comment (h1alexbel) [inline src/main/java/io/github/eocqrs/kafka/settings/BootstrapServers.java:31]: class must be final

**Review by h1alexbel — CHANGES_REQUESTED** [review]
All classes must be final

**Review by h1alexbel — APPROVED** [review]
So far, so good

**Comment by l3r8yJ** [comment]
@h1alexbel a couple of tests, and we will be able to merge

**Comment by h1alexbel** [comment]
@l3r8yJ by the way so you have some ideas how we integrate these objects with existing KfConsumerSettings/ KfProsucerSettings that construct real consumers and producers?

**Comment by l3r8yJ** [comment]
@h1alexbel yes, I'll send a pull request a bit later and you'll tell me what you think

**Comment by h1alexbel** [comment]
@l3r8yJ ok

**Comment by l3r8yJ** [comment]
@h1alexbel take a look, please

**Comment by h1alexbel** [comment]
@rultor merge

**Comment by h1alexbel** [comment]
@l3r8yJ cheers

**Pull request merged**